docs/archive/v5.1/static/ds6/skin-large.css

```diff
@@ -0,0 +1,6 @@
+/*!
+Skin v3.3.1
```

Comment:
I was confused why this version number was wrong, but it turns out this is an empty, redundant file. We can clean this up another time.

Reply:
Yeah. 😉 I saw it a while ago, but actually forgot to delete it.